.eslintrc.yaml

```diff
@@ -76,47 +91,74 @@ rules:
   # of Array#reduce.  Too noisy.
   no-param-reassign: off
 
-  # Imports
+
+  #
+  # Imports; plugin `import`.
+  #
+
   import/prefer-default-export: off
+
   import/no-extraneous-dependencies:
   - error
   - devDependencies: ['**/__tests__/**/*.js', tools/**]
   no-restricted-imports:
   - error
   - patterns: ['**/__tests__/**']
+
   import/no-cycle: off  # This would be nice to fix; but isn't easy.
 
-  # Jest
+
+  #
+  # Jest; plugin `jest`.
+  #
+
```

Comment:
This doesn't really look like it's supposed to be a sort of section header.

I'd suggest something like

```
# ===================================================================
# Jest

# This rule could be useful if...
```

or, with somewhat more brevity,

```
####
# Jest

# This rule could be useful if...
```

or even the Markdown-style

```
# ## Jest
#
# This rule could be useful if...
```

... although personally I'd include the `#` in all cases, not just the final one.